tests/subsys/dfu/dfu_target_stream/overlay-store-progress.conf

```diff
@@ -8,3 +8,8 @@ CONFIG_DFU_TARGET_STREAM_SAVE_PROGRESS=y
 CONFIG_SETTINGS=y
 CONFIG_FLASH_MAP=y
 CONFIG_NVS=y
+
+# Some boards don't have a 'storage' partition defined in their devicetree
```

Comment:
> Some boards

Should ve maybe create a third entry in `tests/subsys/dfu/dfu_target_stream/testcase.yaml` so that we don't need to set `CONFIG_PM_SINGLE_IMAGE=y` for all boards ?

Note: I do see the qemu platforms are already excluded in the tests.

Reply:
I changed the allowed set of platforms, so this is now removed.